temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -349,6 +349,11 @@ message DeleteApiKeyResponse {
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
 
+message GoogleGroupGetUserGroupsFilter {
```

Comment:
Not the biggest fan of this name (I'd just have made a nested `GoogleGroupFilter` in `GetUserGroupsRequest`) but not that big of a deal